sites/public/src/components/listings/search/ListingsSearchCombined.tsx

```diff
@@ -59,35 +72,120 @@ function ListingsSearchCombined(props: ListingsSearchCombinedProps) {
   // This can be changed later if needed
   const pageSize = 25
 
-  const search = async (params: ListingSearchParams, page: number) => {
-    const qb = generateSearchQuery(params)
-    const result = await searchListings(qb, pageSize, page, listingsService)
+  const search = async (page: number, changingFilter?: boolean) => {
+    // If a user pans over empty space, reset the listings to empty instead of refetching
+    if (visibleMarkers?.length === 0 && !changingFilter) {
+      setSearchResults({
+        listings: [],
+        markers: searchResults.markers,
+        currentPage: 0,
+        lastPage: 0,
+        totalItems: 0,
+      })
+      setIsLoading(false)
+      return
+    }
+
+    const modifiedParams: ListingSearchParams = {
```

Comment:
Search the listings by both the filter & the visible markers - but search the markers by only the filter.